src/InstrumentationEngine/ProfilerManagerWrapper.h

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ProfilerManager.h"
+
+namespace MicrosoftInstrumentationEngine
+{
+    class CProfilerManagerWrapper :
```

Comment:
I know that we've been calling it CProfilerManagerWrapper, but maybe we can come up with a better name, e.g. CProfilerManagerForInstrumentationMethod, that better describes for what it is used.